gui/app/controls/ScopeView.qml

```diff
@@ -1,90 +1,164 @@
+        anchors {
+            top: parent.top; topMargin: 8
+            left: parent.left; leftMargin: 22
+        }
+
+        font: Style.theme.font.graphLabel
+        horizontalAlignment: Text.AlignLeft
+        text: yMax.toFixed(0).toString()
+        color: Style.theme.color.textAlternative
     }
-    onHeightChanged:
-    {
-        lineSeries.width =  Math.sqrt(Math.pow(width, 2) + Math.pow(height, 2)) * .0025
+
+    Text {
+        id: yMinLabel
+        anchors {
+            left: parent.left; leftMargin: 22
+            top: yMaxLabel.bottom; topMargin: 90
+        }
+        font: Style.theme.font.graphLabel
+        horizontalAlignment: Text.AlignLeft
+        text: yMin.toFixed(0).toString()
+        color: Style.theme.color.textAlternative
     }
 
-    ValueAxis
-    {
-        id: timeAxis
-        titleText: "Seconds ago"
-        min: -30
-        max: 0
-        labelsColor: chartView.color
+    Row {
+        anchors {
+            top: parent.top; topMargin: 8
+            left: yMaxLabel.right; leftMargin: 8
+        }
+        spacing: 4
+
+        Text {
+            id: nameLabel
+            color: Style.theme.color.textPrimary
+            font: Style.theme.font.parameterDisplay
+        }
+
+        Text {
+            id: unitLabel
+            textFormat: Text.RichText
+            color: Style.theme.color.textAlternative
+            font: Style.theme.font.parameterDisplay
+        }
     }
 
-    ValueAxis
+    ChartView
     {
-        id: valueAxis
-        min: yMin
-        max: yMax
-        labelsColor: chartView.color
+        id: chartView
+
+        //due to a bug on Qt charts 5.11, which is used at the moment
+        // in Raspiberry OS, plotArea property is marked as read-only
+        // making it impossible to control the drawing area.
+        // This was fixed in Qt 5.13 (maybe even on 5.12). Once we have
+        // the fix we can uncomment the following code as the proper
+        // solution:
+
+//         anchors {
+//            top: parent.top; topMargin: 0
+//            left: parent.left; leftMargin: 48
+//            right: parent.right; rightMargin:8
+//            bottom: parent.bottom; bottomMargin: 8
+//        }
+        //plotArea: Qt.rect(0, 0, chartView.width, chartView.height)
+
+        //meanwhile we have to do guess work and some magic numbers to
+        //increase the size of chartview beyond its parent in order to
+        //get a proper rectangle for the plotArea
+
+        anchors {
+            top: parent.top; topMargin: -30
+            left: parent.left; leftMargin: -20
+            right: parent.right; rightMargin:-40
+        }
+        width: parent.width + 100
+        height: parent.height + 70
+
+        backgroundColor: "transparent"
+        legend.visible: false
+        antialiasing: true
+
+
+        property string name
+
+        ValueAxis
+        {
+            id: timeAxis
+            min: -30
+            max: 0
+            labelsVisible: false
```

Comment:
I think we need the axes and grid to be visible - otherwise it is hard to tell what is the time scale of what's happening or exactly what are the values or their rate of change.

Reply:
I guess the axes and grid are not there on the design. (https://xd.adobe.com/view/5816839b-fd33-41f2-5a21-4b218fb9867f-2e27/screen/42f73f40-2ba1-4ffa-83e3-6ba8b625199f/01-mainscreen-Pressure-support-01)

Can we check with @ThomasSonsalla first and defined the changes there?